src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,82 @@
+package frc.robot.subsystems.intake;
+
+import com.revrobotics.*;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+import frc.robot.utils.units.UnitModel;
+
+public class Intake extends LoggedSubsystem<IntakeLoggedInputs> {
+    private static Intake INSTANCE;
+    private final CANSparkMax motor = new CANSparkMax(Ports.Intake.MOTOR, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final CANSparkMax angleMotor = new CANSparkMax(Ports.Intake.ANGLE_MOTOR, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final SparkMaxPIDController pidController = angleMotor.getPIDController();
+    private final RelativeEncoder encoder = angleMotor.getEncoder();
+    private final UnitModel unitModel = new UnitModel(ConstantsIntake.TICKS_PER_RADIAN);
```

Comment:
If the only way to use the angle motor is with degrees, maybe consider using TICKS_PER_DEGREE instead of ticks per radian

Reply:
You are right, I can divide the number of ticks per spin by 360 and then use it.

Reply:
I need to go over the calculations again. something doesn't seem right.